### PR TITLE
Patricks branch

### PR DESCRIPTION
**Pull request by datumcoder**

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -1,22 +1,73 @@
...
+        />
+      </div>
+      <div
+        className={`card__partBlock card__description--${
+          variant ? variant : "white"
+        }`}
+      >
+        <p className="paragraph__title">Length (Meter/Feet)</p>
+        <p className="paragraph">
+          {value || 0} meters = {[units.meters]} feet | {value || 0} feet ={" "}
```
Comment (RafalMichniuk) [inline src/components/Card/Card.jsx:53]: Q: why are you keeping values inside the array?

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -1,22 +1,73 @@
...
+          {[units.feet]} meters
+        </p>
 
-      <p className="paragraph__title">Volume (Liters/GaIIons)</p>
-      <p className="paragraph">0 liters = 0.000 gallons | 0 gallons = 0.000 liters</p>
+        <p className="paragraph__title">Volume (Liters/GaIIons)</p>
+        <p className="paragraph">
+          {value || 0} liters = {[units.liters]} gallons | {value || 0} gallons
```
Comment (RafalMichniuk) [inline src/components/Card/Card.jsx:59]: same as above

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -1,22 +1,73 @@
...
+          = {[units.gallons]} liters
+        </p>
 
-      <p className="paragraph__title">Mass (Kilograms/Pounds)</p>
-      <p className="paragraph">0 kilos = 0.000 pounds | 0 pounds = 0.000 kilos</p>
+        <p className="paragraph__title">Mass (Kilograms/Pounds)</p>
+        <p className="paragraph">
+          {value || 0} kilos = {[units.kilos]} pounds | {value || 0} pounds ={" "}
```
Comment (RafalMichniuk) [inline src/components/Card/Card.jsx:65]: same as above

File: src/components/Card/Card.jsx (hunk 1 above)
Comment (RafalMichniuk) [inline src/components/Card/Card.jsx:53]: Q: is extra space **{" "}** necessary?

**Review by CheerfulCherry — COMMENTED** [review]
some minor changes need to be done

**Pull request closed without merging**